src/rpm/headers/mod.rs

```diff
@@ -0,0 +1,13 @@
+mod header;
+mod lead;
+mod types;
+
+pub use header::*;
+pub use lead::*;
+pub use types::*;
+
+#[cfg(feature = "signing-meta")]
```

Comment:
I am not sure about this cfg tag here. We use the SignatureHeaderBuilder in our RPMBuilder struct. Does this still work when this is gated off behind a cfg feature?

Reply:
I am going to double check this.